packages/angular/build/src/builders/dev-server/vite-server.ts

```diff
@@ -137,8 +137,13 @@ export async function* serveWithVite(
     process.setSourceMapsEnabled(true);
   }
 
-  // TODO: Enable by default once full support across CLI and FW is integrated
-  browserOptions.externalRuntimeStyles = useComponentStyleHmr;
+  // Enable to support component style hot reloading (`NG_HMR_CSTYLES=0` can be used to disable)
+  browserOptions.externalRuntimeStyles = !!serverOptions.liveReload && useComponentStyleHmr;
+  if (browserOptions.externalRuntimeStyles) {
+    // Preload the @angular/compiler package to avoid first stylesheet request delays.
+    // Once @angular/build is native ESM, this should be re-evaluated.
+    await loadEsmModule('@angular/compiler');
```

Comment:
Is there any harm in not awaiting this, just using it to prime the module cache? It should have an exception handler attached to avoid unhandled promise exceptions, but other than that I think it's fine to ignore this result (although tsetse may disagree)

Reply:
shouldn't be any